src/props.js

```diff
@@ -120,6 +121,7 @@ export const defaultProps = {
     },
     facebook: {
       appId: '1309697205772819'
+      version: 'v2.5'
```

Comment:
Maybe we should bump this to `v3.3` if that's the latest? Does everything work as normal?

```suggestion
      version: 'v3.3'
```

Reply:
I had changed to `v3.3`, and I did not find any problem.